src/components/form/file_picker/_file_picker.scss

```
@@ -115,7 +116,8 @@
 
   // make normal sized prompt look like placeholder
```

Also, this comment is no longer applicable since the text no longer looks like placeholder text (it's blue instead of gray). It's also confusing why the text should change color between blue and black on non-large vs large sizes TBH. Why not just make it primary/blue on all sizes?

---

On the color question, mainly just preference. The large size is quite identifiable as a dropzone since its a pretty standard pattern, the brighter blue text is a little distracting and superfluous. On the small version, though, I needed the extra communication that the element was interactive and clickable.

---

That works for me, but let's please document that in the comment as a deliberate decision!
```suggestion
  // make normal sized prompt stand out a bit more - on large size we don't need this as it's already identifiable
```